### DISCUSSION TITLE
Renewing GitHub Education gets stuck at school selection

### DISCUSSION THREAD

**Discussion by thegrasshopper104** [community/community#131539] [GitHub Education]
### Hi everyone,

Hi,
I applied for GitHub Education benefits previously and got approved.
I'm trying to apply again, but the application form doesn't let me continue after selecting my school. I've made sure to allow location access, but nothing happens after I click on "Continue".
What should I  do?

**Comment by NSerbin** [comment]
I have the same issue and in 1 week ll expire mine too
